browser/ui/views/toolbar/brave_toolbar_view.cc

```diff
@@ -149,6 +149,11 @@ void BraveToolbarView::Init() {
     return;
   }
 
+  if (split_tabs_) {
+    auto split_tabs = container_view->RemoveChildViewT(split_tabs_);
```

Comment:
nit: Could you add some comments why we're removing this?

Reply:
Added.